### PR TITLE
Update changelog and bump version for 0.4.9-rc1

### PR DESCRIPTION
**Pull request by whyrusleeping**
License: MIT
Signed-off-by: Jeromy <jeromyj@gmail.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,32 @@
 # go-ipfs changelog
 
+### 0.4.9 - 2017-04-30
+
+- Features
```
Comment (Kubuxu) [inline CHANGELOG.md]: Those below aren't all features. Do you want to separate them into categories or to just change title to Changes?

Comment (lgierth) [inline CHANGELOG.md]: There's only one, the rest are "Improvements" :)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,32 @@
...
+  - Update go-multihash with keccak and bitcoin hashes ([ipfs/go-ipfs#3833](https://github.com/ipfs/go-ipfs/pull/3833))
+  - Update go-is-domain to contain new gTLD ([ipfs/go-ipfs#3873](https://github.com/ipfs/go-ipfs/pull/3873))
+  - Handle 404 correctly in dist_get tool ([ipfs/go-ipfs#3879](https://github.com/ipfs/go-ipfs/pull/3879))
+  - Periodically flush cached directories during ipfs add ([ipfs/go-ipfs#3888](https://github.com/ipfs/go-ipfs/pull/3888))
+- Documentation
+  - Change issue template to use Severity instead of Priority ([ipfs/go-ipfs#3834](https://github.com/ipfs/go-ipfs/pull/3834))
+  - Fix link to commit hook script in contribute.md ([ipfs/go-ipfs#3863](https://github.com/ipfs/go-ipfs/pull/3863))
+  - Fix install_unsupported for openbsd, add docs ([ipfs/go-ipfs#3880](https://github.com/ipfs/go-ipfs/pull/3880))
+- Bugfixes
+  - Fix wanlist typo in prometheus metric name ([ipfs/go-ipfs#3841](https://github.com/ipfs/go-ipfs/pull/3841))
```
Comment (lgierth) [inline CHANGELOG.md]: I was about to say typo thre, but oh the irony.

Comment (whyrusleeping) [inline CHANGELOG.md]: lol, I figured it could stay...

Comment (Kubuxu) [inline CHANGELOG.md]: You see why it happened.

Comment (Kubuxu) [inline CHANGELOG.md]: but it wasn't me this time, and I think it was on purpose

Comment (lgierth) [inline CHANGELOG.md]: Yeah I figured it was on purpose ;) can stay

**Comment by whyrusleeping** [comment]
@lgierth the cache control one is in there, and i left out the restoring commands order because it was fixing a previous pull request, so the net change from the previous release is zero.

**Comment by lgierth** [comment]
> the cache control one is in there

Oh man I should read these in rendered mode...

> left out the restoring commands order because it was fixing a previous pull request, so the net change from the previous release is zero

fair :+1:

**Comment by whyrusleeping** [comment]
@lgierth cool, and i fixed the placement of the dist_get thing

**Pull request merged**